webview/src/shared/components/dragDrop/DragDropContainer.tsx

```diff
@@ -189,22 +195,30 @@ export const DragDropContainer: React.FC<DragDropContainerProps> = ({
     if (isSameGroup(draggedRef?.group, group)) {
       const { id } = e.currentTarget
       !disabledDropIds.includes(id) &&
-        id === draggedOverId &&
+        id !== draggedId &&
         setDirection(getDragEnterDirection(e))
     }
   }
 
+  const handleDragEnd = () => {
+    cleanup()
+    setDraggedRef?.(undefined)
+  }
+
   const buildItem = (id: string, draggable: JSX.Element) => (
     <draggable.type
       key={draggable.key}
       {...draggable.props}
       onDragStart={handleDragStart}
-      onDragEnd={cleanup}
+      onDragEnd={handleDragEnd}
       onDragOver={handleDragOver}
       onDragEnter={handleDragEnter}
       onDrop={handleOnDrop}
       draggable={!disabledDropIds.includes(id)}
-      style={(id === draggedId && { display: 'none' }) || draggable.props.style}
+      style={
+        (id === draggedId && hideDragged && { display: 'none' }) ||
+        draggable.props.style
+      }
```

Comment:
I see it was like this before this PR, but is the value of this style prop not just a weirdly written ternary? i.e. `id === draggedId && hideDragged ? {display: 'none'} : draggable.props.style`

Reply:
Not sure in this particular case, but I think our linter favours this syntax over ternaries using `?` and `:` sometimes.

Reply:
Agreed. I changed it to a ternary and ESLint seemed to be fine with it in this case.